src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/UpdateApprenticeshipStatus/UpdateApprenticeshipStatusCommandHandler.cs

```diff
@@ -52,6 +60,9 @@ public UpdateApprenticeshipStatusCommandHandler(IEmployerCommitmentApi commitmen
             await ValidateDateOfChange(command, validationResult);
 
             await _commitmentsApi.PatchEmployerApprenticeship(command.EmployerAccountId, command.ApprenticeshipId, apprenticeshipSubmission);
+
+            var apprenticeship = await _commitmentsApi.GetEmployerApprenticeship(command.EmployerAccountId, command.ApprenticeshipId);
+            await _providerEmailNotificationService.SendProviderApprenticeshipStopNotification(apprenticeship);
```

Comment:
Shouldn't we check that 'command.ChangeType == ChangeStatusType.Stop' before sending a stop notification?